test.js

```diff
@@ -84,5 +95,18 @@ describe('<message>', () => {
       const parsed = parser('fix: address major bug\n\nthis is the first line of the body\n\nthis is the second line of body\n\nAuthor: @bcoe\nRefs #392')
       parsed.should.matchSnapshot()
     })
+    it('contains valid positions for footer after multi-line body', () => {
+      assertNodePositions('fix: address major bug\n\nthis is the first line of the body\n\nthis is the second line of body\n\nAuthor: @bcoe\nRefs #392')
+    })
   })
 })
+
+function assertNodePositions (text) {
+  visit(parser(text), node => {
+    if (node.children) return
+    const valueByPoint = source(node, text)
```

Comment:
For nodes where we've dropped whitespace information, will `source` fail today? This seems like a much better approach for getting the original header than I'd used in #19, perhaps I can move to this approach after we land this?

Reply:
As long as we don't include that in the positional information, should be fine. I'm not sure if that's the correct approach, we still remove data from the original content. But, that piece of info is not part of that node.